snap/snapcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: envtester
+base: core24
+version: 'demo'
+summary: Demonstration of env-exporter program
+description: |
+  envtester is a demo snap to show the env-exporter program usecases
+
+platforms:
+  amd64:
+    build-on:  [amd64]
+    build-for: [amd64]
+  arm64:
+    build-on:  [arm64]
+    build-for: [arm64]
+  riscv64:
+    build-on:  [riscv64]
+    build-for: [riscv64]
+  arm64-cross:
+    build-on:  [amd64]
+    build-for: [arm64]
+  riscv64-cross:
+    build-on:  [amd64]
+    build-for: [riscv64]
+
+grade: stable
+confinement: strict
+
+apps:
+  envtester:
+    plugs: [ home ]
+    command-chain: [ bin/env-exporter ]
+    command: envtester
```

Comment:
To test the global vs local:
- Change this to a oneshot service
- Replicate to have two services with different names
- Call them myapp1 and myapp2 for clarity?
- Restart the apps every time some new envs have been set